Orange/widgets/data/owdiscretize.py

```diff
@@ -291,7 +289,7 @@ def _reset(self):
         # restore the individual variable settings back to defaults.
         def_method = self._current_default_method()
         self.var_state = {}
-        for i in range(len(self.varmodel)):
+        for i, _ in enumerate(self.varmodel):
```

Comment:
Why not `range(len(...))`? I think it was better. If lint complains, can we fix it elsewhere (or silence it)?